src/vec.ts

```diff
@@ -86,35 +86,31 @@ export namespace vec {
    * Multiply two matrices (compose 2D transformations)
    */
   export function matrixMult(m: Matrix, m2: Matrix): Matrix {
-    return [
-      m[0] * m2[0] + m[1] * m2[3] + m[2] * m2[6],
-      m[0] * m2[1] + m[1] * m2[4] + m[2] * m2[7],
-      m[0] * m2[2] + m[1] * m2[5] + m[2] * m2[8],
-      m[3] * m2[0] + m[4] * m2[3] + m[5] * m2[6],
-      m[3] * m2[1] + m[4] * m2[4] + m[5] * m2[7],
-      m[3] * m2[2] + m[4] * m2[5] + m[5] * m2[8],
-      m[6] * m2[0] + m[7] * m2[3] + m[8] * m2[6],
-      m[6] * m2[1] + m[7] * m2[4] + m[8] * m2[7],
-      m[6] * m2[2] + m[7] * m2[5] + m[8] * m2[8],
-    ]
+    return matrixCreate(
```

Comment:
I made this change by find-replacing each of the `[6]` and `[7]` entries with `0` and the `[8]` entry with 1, then manually simplifying :)